docs/source/frameworks/transformers.rst

```diff
@@ -87,6 +87,50 @@ such as tokenizers, preprocessors, and feature extractors, can also be saved as
 
 To load the pre-trained instances for testing and debugging, use :code:`bentoml.transformers.load_model` with the same tags.
 
+Starting from BentoML version 1.1.9, importing pre-trained Transformer models from Hugging Face has been further streamlined.
+You can choose to use the new ``bentoml.transformers.import_model`` function to import models directly into the BentoML Model Store without the overhead of loading them into memory.
+By contrast, ``bentoml.transformers.save_model`` necessitates prior model loading and can be resource-intensive for models with large weights.
+Here is an example of using the new function:
+
+.. code-block:: python
+    :caption: `download_model.py`
+
+    import bentoml
+    from transformers import AutoTokenizer
+
+    # Save the tokenizer with minimal memory overhead
+    tokenizer = AutoTokenizer.from_pretrained("t5-small")
+    bentoml.transformers.save_model('t5-small-tokenizer', tokenizer)
+
+    # Import the model without loading into memory, conserving memory
+    bentoml.transformers.import_model("t5-small-model", "t5-small")
+
+The ``bentoml.transformers.import_model`` function has two required parameters:
+
+* ``name``: The name of the model in the BentoML Model Store.
+* ``model_name_or_path``: This can be a string, a Hugging Face repository identifier (repo_id), or a directory path containing weights saved using ``transformers.AutoModel.save_pretrained`` (for example, ``./my_pretrained_directory/``).
+
+When importing models from repositories that require the keyword argument ``trust_remote_code=True`` for custom pipeline execution, BentoML will load the model into memory by default.
+In such cases, to avoid loading the model into memory, add the keyword argument ``clone_repository=True``. This downloads all the files in the repository instead of selectively picking certain model files,
+resulting in greater storage requirements; this approach uses the ``.from_pretrained`` method under the hood, avoiding any potential memory usage of the model weights. Here is how you can invoke this:
```

Comment:
oops, sorry I might have flipped the things I said before. 

Using `clone_repository=True` actually avoids the use of `from_pretrained`, that's why we can avoid the memory usage. `clone_repository=False` will use `from_pretrained` by default and this will normally not load into memory except for when `trust_remote_code=True`. 

Sorry about my wordy explanation beforehand.

Reply:
OK. I think we can just removed the `from_pretrained` explanation part.